Updates support (#2216)
Azure/data-api-builder: SUPPORT.md

# Support

Data API builder is an open-source project with contributions from Microsoft and the community to build a configuration-based engine in a Docker-friendly container that easily adds a secure and feature-rich Data API over databases. Data API builder is not a Microsoft product and, as a result, is not supported by Microsoft as a product would be.

## How to file issues and get help

### GitHub Issues

> Engineers who contribute to and work on Data API builder periodically monitor and review issues added to [the repository's issues list](https://github.com/Azure/data-api-builder/issues). The period of review is not set, unlike service level agreements (SLA) with Microsoft products, and resolution is not guaranteed. When an issue is raised, a member of the team will respond as soon as possible; please be patient.

Reviewer: ```suggestion
Engineers who contribute to and work on Data API builder periodically monitor and review issues added to [the repository's issues list](https://github.com/Azure/data-api-builder/issues). The period of review is not set by a service level agreement (SLA), and resolution is not guaranteed. When an issue is raised, a member of the team will respond as soon as possible; please be patient.
```

> Repository issues track bugs, questions, and feature requests. Creating issues requires a GitHub account; templates help ensure information is used for investigation. Repository issues may be added to the backlog by engineers and become triaged in our future roadmap.

Reviewer: ```suggestion
Repository issues track bugs, questions, and feature requests. Creating issues requires a GitHub account; issue templates prompt for details that will help engineers reproduce your issue and more effectively complete an investigation. Repository issues may be added to the backlog by engineers and become triaged in our future roadmap.
```

We recommend searching for existing issues similar to yours before creating a new one. Commenting on existing issues helps coalesce recurring issues and helps engineers evaluate severity and find better resolutions.

### Before Submitting an Issue

First, please do a search in [open issues](https://github.com/Azure/data-api-builder/issues) to see if the issue or feature request has already been filed. Use this [query](https://github.com/Azure/data-api-builder/issues?q=is%3Aopen+is%3Aissue+sort%3Areactions-%2B1-desc+label%3Aenhancement+) to search for the most popular feature requests.
If you find your issue already exists, make relevant comments and add your [reaction](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments). Use a reaction in place of a "+1" comment.

:+1: - upvote

:-1: - downvote

> ### Azure services support tickets

Reviewer 2: ```suggestion
### Azure services support tickets

```

nit: leave a line after subtitle like others.

If your solution hosts Data API builder containers using Azure Static Web Apps, Azure Container Apps, Azure Container Instances, Azure Kubernetes Services, or Azure Web Apps for Containers, then the cause of your issue could be that service. If you wish to open a support ticket with that service, do so in your [Azure portal](https://learn.microsoft.com/en-us/azure/azure-portal/supportability/how-to-create-azure-support-request). 

> ### Azure database support tickets

Reviewer 2: ```suggestion
### Azure Database support tickets
```

If your solution uses an Azure database, including Azure SQL, Azure Cosmos DB, Azure Database for PostgreSQL, Azure Database for MySQL, and Azure SQL Data Warehouse, then the cause of your issue could be with that database. If you wish to open a support ticket with that database, do so in your [Azure portal](https://learn.microsoft.com/en-us/azure/azure-portal/supportability/how-to-create-azure-support-request).

> Data API builder containers using Azure Static Web Apps, Azure Container Apps, Azure Container Instances, Azure Kubernetes Services, or Azure Web Apps for Containers, then the cause of your issue could be that service. If you wish to open a support ticket with that service, do so in your [Azure portal](https://learn.microsoft.com/en-us/azure/azure-portal/supportability/how-to-create-azure-support-request). 

Reviewer 2: how is this paragraph 

> Data API builder containers using Azure Static Web Apps, Azure Container Apps, Azure Container Instances, Azure Kubernetes Services, or Azure Web Apps for Containers, then the cause of your issue could be that service. If you wish to open a support ticket with that service, do so in your [Azure portal](https://learn.microsoft.com/en-us/azure/azure-portal/supportability/how-to-create-azure-support-request).

different than the content above `Azure services support tickets`

Author: It isn't a different link, just a different context to get there. I think this is okay.
